XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1472,6 +1472,29 @@
       "ImagePath": "",
       "Description": "",
       "Items": [
+        {
+          "UniqueId": "AnimatedVisualPlayer",
+          "Title": "AnimatedVisualPlayer",
+          "Subtitle": "The AnimatedVisualPlayer allows playback of animated Windows.UI.Composition.Visual trees.",
+          "ImagePath": "Assets/Animations.png",
+          "Description": "The AnimatedVisualPlayer allows playback of animated Windows.UI.Composition.Visual trees.",
+          "Content": "<p>Look at the <i>AnimatedVisualPlayerPage.xaml</i> and <i>AnimatedVisualPlayerPage.xaml.cs</i> files in Visual Studio to see the full code for this page.</p>",
+          "IsNew": true,
```

Comment:
Is this control still in preview in WinUI, or has it formally shipped? If it's still in preview, we need to mark it differently.

Reply:
It's in preview today but will be released whenever @jesbis has the next release of WinUI planned. All the code for AVP has been migrated to the release branch.